docs/setup/asb/systemd/restart-asb.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Restart ASB service
+After=watch-asb-binary-change.path
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/systemctl restart start-asb.service
+ExecStopPost=/bin/sh -c 'if [ "$$EXIT_STATUS" -ne 0 ]; then /usr/local/bin/systemd-email.sh daniel@coblox.tech %n; fi'
```

Comment:
That should just be an `OnFailure` no?

Reply:
Yeah, seems I should have digged a little deeper. 

`OnFailure` is an attribute of the `Unit` section, not the `Service` section. `OnFailure` can only trigger another service, so in order to achieve this behavior I'd have to configure a notification service similar to what is described here: https://serverfault.com/questions/694818/get-notification-when-systemd-monitored-service-enters-failed-state

I will do that.

Reply:
Added an additional templated service `unit-status-email@.service` that can be triggered on `OnFailure` by any other service like this: `OnFailure=unit-status-email@%n.service`

Where `%n` is the full service name of the caller (passed as parameter to `unit-status-email@.service`).